sample/src/main/java/com/jintin/clerk/sample/MainActivity.kt

```diff
@@ -23,19 +23,35 @@ class MainActivity : AppCompatActivity() {
         })
 
         model.getTickEvent().observeForever {
-            ClerkUtils.log(channelText.text.toString(), logText.text.toString() + it)
+            ClerkUtils.d(channel = channelText.text.toString(), text = logText.text.toString() + it)
 //            // This one also work
-//            ClerkUtils.log(this, channelText.text.toString(), logText.text.toString() + it)
+//            ClerkUtils.d(this, channelText.text.toString(), logText.text.toString() + it)
         }
 
         tick.setOnClickListener {
             model.toggleTick()
         }
 
         send.setOnClickListener {
-            ClerkUtils.log(channelText.text.toString(), logText.text.toString())
+            ClerkUtils.e(channel = channelText.text.toString(), text = logText.text.toString())
 //            // This one also work
-//            ClerkUtils.log(this, channelText.text.toString(), logText.text.toString())
+//            ClerkUtils.e(this, channelText.text.toString(), logText.text.toString())
+        }
+
+        verbose.setOnClickListener {
+            ClerkUtils.v(channel = "VERBOSE:", text = "TEST_VERBOSE")
```

Comment:
Type and channel can be mean different things, no need to combine this concept together?